src/Microsoft.Extensions.Configuration.UserSecrets/UserSecretsIdAttribute.cs

```diff
@@ -6,15 +6,15 @@
 namespace Microsoft.Extensions.Configuration.UserSecrets
 {
     /// <summary>
-    /// Represents the user secrets id.
+    /// Represents the user secrets ID.
```

Comment:
Seems like a good idea. `.AddUserSecrets` is probably the best place for this since that is the only API most samples/docs use.